src/mame/dataeast/karnov.cpp

```diff
@@ -633,6 +636,13 @@ void karnov_state::playfield_w(offs_t offset, u16 data, u16 mem_mask)
 	m_bg_tilemap->mark_tile_dirty(offset);
 }
 
+void karnov_state::vintctl_w(offs_t offset, u16 data, u16 mem_mask)
+{
+	m_vint_en = (offset & 1)!=0;
+	// writting to any position in the range will clear the line
```

Comment:
`m_vint_en = bool(offset & 1);` should work, also `writting`

Reply:
Isn't `(offset & 1)!=0` a bool type already?